README.md

```diff
@@ -1,89 +1,162 @@
-SYCL-BLAS Implementation
+# SYCL-BLAS, a SYCL Basic Linear Algebra Library 
 ===
 
-SYCL-BLAS implements BLAS - [Basic Linear Algebra Subroutines](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprogram) - using [SYCL 1.2](
-https://www.khronos.org/registry/sycl/specs/sycl-1.2.pdf), the
-[Khronos](http://www.khronos.org) abstraction layer for [OpenCL](https://www.khronos.org/opencl/).
+The SYCL-BLAS library is an implementation of the 
+[Basic Linear Algebra Subroutines](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprogram) - using [SYCL 1.2.1](
```

Comment:
This is a stylistic thing, but I would prefer if all the links in markdown are provided at the end rather than inline.

e.g.
```markdown
blah blah [link text] blah blah [some really long link text][short link] blah
...

[link text]: https://actual_link.com
[short link]: https://other_link.com
```